### ISSUE TITLE
Workspace php56

### ISSUE THREAD

**Issue by winfried-van-loon** [laradock/laradock#748]
Currently, the workspace Dockerfile outputs a bunch of errors when installing using env `PHP_VERSION` set to `56`, I think most of them have to do with the paths used, however I don't know how to correct them. One of the first ones to fail is (by enabling it in the .env first, of course):

https://github.com/laradock/laradock/blob/master/workspace/Dockerfile-56#L64

```
echo "extension=soap.so" >> /etc/php/5.6/cli/conf.d/40-soap.ini \
```

**Comment by winfried-van-loon** [comment]
There's a separate issue open for the PHP version within the workspace: #776. Linking everything together so we can find all the related issues when the solution is available.

**Comment by mikeerickson** [comment]
@winfried-van-loon With all these PRs, I am a bit confused as to what is what.   Can you please summarize for me before I go through the process of the merge process (this is going to take a decent amount of testing to make sure all the pieces fall in line)

**Comment by winfried-van-loon** [comment]
@mikeerickson Yeah, I can imagine. I haven't had the time to test them all yet myself. Currently I'm at work, so haven't had the time yet to write proper descriptions.

All but two have a related issue number in the pull requests, which contain the related discussion. Most of the pull requests are in existing code and don't have many changes each. At a later time, I'll do more testing and write a better description to the open pull requests.

**Comment by mikeerickson** [comment]
@winfried-van-loon Sounds good, I appreciate it.  I too am at work at the moment, but will try and get these reviewed and merged in next couple days.

**Comment by winfried-van-loon** [comment]
This issue is waiting on laradock/workspace#15

**Comment by winfried-van-loon** [comment]
Solved in base image (requires upgrade and re-build). 👍
